Bonsai.Tinkerforge/BrickletGPSV2.cs

```diff
@@ -0,0 +1,85 @@
+﻿using System;
+using System.ComponentModel;
+using System.Reactive.Disposables;
+using System.Reactive.Linq;
+using Tinkerforge;
+
+
+namespace Bonsai.Tinkerforge
+{
+    public class BrickletGPSV2 : Combinator<IPConnection, BrickletGPSV2.CoordinateData>
+    {
+        [Description("The unique bricklet device UID.")]
+        public string Uid { get; set; }
+
+        [Description("Specifies the period between sample event callbacks. A value of zero disables event reporting.")]
+        public long Period { get; set; } = 1000;
+
+        [Description("If SBAS is enabled, the position accuracy increases (if SBAS satellites are in view), but the update rate is limited to 5Hz. With SBAS disabled the update rate is increased to 10Hz.")]
+        public SBASConfig SBAS { get; set; } = SBASConfig.Disabled;
+
+        [Description("Specifies the behavior of the status LED.")]
+        public BrickletGPSV2StatusLedConfig StatusLed { get; set; } = BrickletGPSV2StatusLedConfig.ShowStatus;
+
+        // TODO - can we get all the GPS data structures output here? GetStatus, GetAltitude, GetMotion, GetDateTime, GetSatelliteSystemStatus - could generate several observables for each Get function and then Merge
```

Comment:
This is a very good point. I guess there are several options here, but to simplify we could check if it is possible to configure the period callbacks for all streams to the same value, and if they are reliably reported from the device. If this is the case, I guess we could `Zip` the outputs of all streams together and produce a combined DataFrame with all the data.

Another option is to have further operators downstream that select the individual callbacks, much as we do for `CreateIPConnection` and the individual bricklets.

Reply:
See [c233a72]

Reply:
Yes, I can see how that would happen. I think we would rather have just one GPS operator that you can then store in a subject somewhere and then pull the different streams on demand. Let's discuss a bit about this offline and we can converge on a solution.